content/docs/self-hosted/overview.md

```diff
@@ -1,12 +1,161 @@
 ---
 title: Self Hosted Tina
-last_edited: '2023-02-01T04:00:00.000Z'
+last_edited: '2023-07-06T07:00:00.000Z'
+next: /docs/self-hosted/starters/nextjs-vercel
 ---
 
-TinaCMS treats the filesystem as the single source of truth for you content but still requires the Tina Data Layer to do its job. [Tina Cloud](/docs/tina-cloud/) runs the Tina Data Layer for you and also handles user authentication and communication with GitHub.
+## Introduction
 
-For users who do not wish to rely on Tina Cloud, **we also offer a self-hosted alternative** where you can host your own Tina Data Layer and provide your own user authentication and GitHub connection. We love the control and portability you get when storing content in Markdown and JSON files and want to extend that to the rest of the CMS.
+By default, TinaCMS uses Tina Cloud as its backend. Tina Cloud is a powerful, out-of-the-box solution that handles reading/writing to your GitHub repository, caching content in a queryable data layer, and authentication / authorization.
 
-If you're interested in self-hosting Tina, you can read about this solution on [GitHub](https://github.com/tinacms/tinacms/discussions/3589) or in this [blog post](https://tina.io/blog/self-hosted-datalayer/).
+For users who want to be independent of Tina Cloud, **we also offer a self-hosted alternative** where you can host your own Tina Data Layer and provide your own user authentication and git integration. We love the control and portability you get when storing content in Markdown and JSON files and this allows users to extend that flexibility to the rest of the CMS.
 
-**Note on Licensing -** Most of TinaCMS is licensed under the Apache 2.0 license, but the Tina Data Layer is released under a modified version of the Fair Source License. You can find this license [here](https://github.com/tinacms/tinacms/blob/main/packages/@tinacms/datalayer/LICENSE) and its corresponding FAQ, [here](https://github.com/tinacms/tinacms/blob/main/packages/@tinacms/datalayer/LICENSE-FAQ.md).&#x20;
+
+## How does it work?
+
+You create a Tina Data Layer by creating backend functions that host the GraphQL API and you create a Data Layer that indexes content from the filesystem into a databse. This database is used to retrive the data. When an editor updates the data a [git provider](/docs/self-hosted/git-provider/overview/) is used to commit the changes to the filesystem.
+
+## Getting Started
+
+### Using a Example (Recommended)
+
+Self hosting TinaCMS can be a bit of a challenge. We have created a [example](/docs/self-hosted/starters/overview/) that you can use to get started. This starter is a Next.js app that uses Vercel for hosting and GitHub for git integration. It also includes Next Auth authentication.
+
+
+### Using the CLI init command
```

Comment:
Should we move this to its own doc? It feels a bit weird that our recommended approach of using the starter links out to another doc, but this section doesn't.